skimage/feature/_hog.py

```diff
@@ -276,6 +282,9 @@ def hog(image, orientations=9, pixels_per_cell=(8, 8), cells_per_block=(3, 3),
 
     n_blocks_row = (n_cells_row - b_row) + 1
     n_blocks_col = (n_cells_col - b_col) + 1
+    if n_blocks_col <= 0 or n_blocks_row <= 0:
+        raise ValueError('The image given is not large enough for the cells_per_block '
+                         'and pixels_per_cell given')
```

Comment:
What do you think about telling to the user in the same error message what should be the minimal size of the image given the provided `cells_per_block` and `pixels_per_cell`?

Reply:
I agree that would be more clear, I have updated this accordingly.